stock_picking_value_triple_discount/report/stock_valued_picking_triple_discount.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="valued_report_picking" inherit_id="stock.report_delivery_document">
```

Comment:
shouldn't the `inherit_id` be `stock_valued_picking_report.valued_report_picking`?

Reply:
You are right, changes done